srcpkgs/Komikku/template

```diff
@@ -1,20 +1,21 @@
 # Template file for 'Komikku'
 pkgname=Komikku
-version=1.19.0
+version=1.21.1
 revision=1
 build_style=meson
+build_helper=gir
 hostmakedepends="gettext glib-devel gobject-introspection pkg-config
  desktop-file-utils gtk-update-icon-cache"
 makedepends="gtk4-devel libadwaita-devel"
 depends="gtk4 libadwaita libnotify libsecret python3-BeautifulSoup4 python3-Brotli
  python3-Pillow python3-Unidecode python3-requests python3-dateparser
  python3-gobject python3-keyring python3-lxml python3-magic python3-rarfile
  python3-natsort python3-pure-protobuf python3-emoji libwebkitgtk60
- python3-piexif"
+ python3-piexif bubblewrap xdg-dbus-proxy"
```

Comment:
why is this necessary?

Reply:
I think this is a problem with your setup and should be left to you to solve. I don't see any mention of bubblewrap in the source code.

Reply:
Reverted.